### PR TITLE
Clarify end-to-end security language

### PR DESCRIPTION
**Pull request by bifurcation**

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -37,12 +37,17 @@ decryption of media payloads where required.
...
-end-to-end encrypted for certain usecases. The keying mechanisms for media 
-confidentiality is however outside the scope of this working group. Even when media is
-end-to-end encrypted, the relays can access metadata needed for caching
-(such as timestamp), making media forwarding decisions (such as drop or
-delay under congestion) and so on.
+to improve the delivery performance.
+
+Media will be encrypted at the transport layer using the standard QUIC
+mechanisms.  Media content may be end-to-end encrypted in certain use cases,
+where the "end-to-end" keys are avialable to media sources and consumers, but
```
Comment (ekr) [inline charter.md:44]:
```suggestion
where the "end-to-end" keys are available to media sources and consumers, but
```

**Review by ekr — COMMENTED** [review]
This is pretty similar to my PR. I could go with either one.

**Comment by bifurcation** [comment]
Ah, hadn't seen your PR @ekr.  This one seems a little stronger, in that it requires metadata authentication.  But I could live with either.

File: charter.md [hunk 2]
```diff
@@ -37,12 +37,17 @@ decryption of media payloads where required.
...
+not relays.  Even when media is end-to-end encrypted, the relays can access
+metadata needed for caching (such as timestamp), making media forwarding
+decisions (such as drop or delay under congestion) and so on.  Such media
```
Comment (martinthomson) [inline charter.md:47]:
```suggestion
decisions (such as drop or delay under congestion), and so on.  Such media
```

"and so on" is a bit weak, but I don't have a suggestion here.  How we decide what is OK for a relay to see is important, almost as important as what we decide is OK.

**Review by martinthomson — APPROVED** [review]

File: charter.md (hunk 2 above)
Comment (bifurcation) [reply charter.md:47]: I didn't change that text :)  I'm a little split -- on the one hand, the choice of metadata is clearly important for privacy, but on the other hand, there's not much we can do to constrain things.

Comment (suhasHere) [reply charter.md:47]: i think we need to clarify that by saying 
for relays to make drop/forward/store decisions and effectively react to congestion feedback from the underlying QuiC Transport.

Comment (suhasHere) [reply charter.md]: @martinthomson would that work ?

**Comment by hardie** [comment]
Based on discussion in BoF, approving this PR, and will reject ekr's similar PR.

**Pull request merged**